internal/controller/erdmadevice_controller.go

```diff
@@ -68,28 +68,30 @@ func (r *ERdmaDeviceReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 	}
```

Comment:
If localERIDiscovery is enabled, the controller is not required and should not be installed; only the agent needs to run on each node. This simplifies deployment and avoids unnecessary components.